python/src/worker/tasks/trajectory_graph.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+import backoff
+import requests
+from aws_xray_sdk.core import xray_recorder
+
+from ..config import config
+from ..result import Result
+from ..tasks import Task
+
+
+class GetTrajectoryGraph(Task):
+    def _format_result(self, raw):
+        # JSONify result.
+        raw_result = json.dumps(raw)
+
+        # Return a list of formatted results.
+        return [Result(raw_result), Result(raw_result)]
+
+    @xray_recorder.capture("GetTrajectoryGraph.compute")
+    @backoff.on_exception(
+        backoff.expo, requests.exceptions.RequestException, max_time=30
+    )
+    def compute(self):
+        request = {}
```

Comment:
Doesn't the worker need to take the embedding passed from the UI?

Reply:
Embedding is already contained inside the Seurat object, so we don't need to pass it in again.